### PR TITLE
make quickstart drawer resizable

### PR DESCRIPTION
**Pull request by rottencandy**
**Story**: 
https://issues.redhat.com/browse/ODC-5444

**Description**: 
Make quick start drawer resizable, and undo changes made in #7607 so that drawer is resizable on smaller screens

**Screen shots / Gifs for design review**: 

https://user-images.githubusercontent.com/20013884/107213953-30394980-6a2f-11eb-8d06-af17c55a59a8.mp4

**Browser conformance**: 

- [x] Chrome
- [x] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

**Comment by debsmita1** [comment]
cannot resize the QS drawer on firefox, the drawer covers almost the entire screen
works fine on chrome

**Comment by debsmita1** [comment]
> cannot resize the QS drawer on firefox, the drawer covers almost the entire screen
> works fine on chrome

updated my firefox to 85.0.2 (64-bit)
works fine now

**Comment by rottencandy** [comment]
> cannot resize the QS drawer on firefox, the drawer covers almost the entire screen
> works fine on chrome

This appears to be a PatternFly bug, on Firefox 72, raised a bug for it: https://github.com/patternfly/patternfly-react/issues/5437

**Review by rohitkrai03 — COMMENTED** [review]
@rottencandy We should also look at upgrading PF package to include latest bug fixes related to the drawer. There were quite a few bugs open last time when we discussed about this.

**Comment by rottencandy** [comment]
> @rottencandy We should also look at upgrading PF package to include latest bug fixes related to the drawer. There were quite a few bugs open last time when we discussed about this.

Looks like we are already using the latest version of the [@patternfly/react-core](https://www.npmjs.com/package/@patternfly/react-core) package, which is `4.90.2`.

File: frontend/package.json [hunk 1]
```diff
@@ -150,7 +150,7 @@
     "null-loader": "^3.0.0",
     "openshift-logos-icon": "1.7.1",
     "patternfly": "^3.59.5",
-    "patternfly-react": "2.39.16",
+    "patternfly-react": "2.39.17",
```
Comment (rohitkrai03) [inline frontend/package.json:153]: I think this is the older PF 3 react package. The new one exists in `@patternfly/react-core`.

**Review by rohitkrai03 — CHANGES_REQUESTED** [review]

File: frontend/package.json (hunk 1 above)
Comment (rottencandy) [reply frontend/package.json]: Seems like we are using the latest available version then.

Comment (rottencandy) [reply frontend/package.json]: Updated `@patternfly/react-core` to 4.97.2, which includes the fix: https://www.patternfly.org/v4/developer-resources/release-notes/html

**Review by rohitkrai03 — APPROVED** [review]
/approve
/lgtm

**Pull request merged**